reframe/frontend/cli.py

```diff
@@ -368,6 +372,24 @@ def main():
     printer.colorize = site_config.get('general/0/colorize')
     printer.inc_verbosity(site_config.get('general/0/verbose'))
 
+    if options.upgrade_config_file is not None:
```

Comment:
To be consistent with the rest:

```suggestion
    if options.upgrade_config_file:
```